src/test-utils.tsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable @typescript-eslint/ban-ts-ignore */
```

Comment:
Codacy found an issue: [Definition for rule '@typescript-eslint/ban-ts-ignore' was not found.](https://app.codacy.com/gh/Wyvarn/space-xplorer-client/pullRequest?prid=6087079)